context.altmovie.posters/capture.py

```diff
@@ -78,12 +78,13 @@ def searchTitle(self, title, dbID):
         for idx in range(1,SEARCHDEPTH):
             try:
                 SEARCHURL = 'http://www.alternativemovieposters.com/page/%d/?s=%s'
-                soup = BeautifulSoup(self.openURL(SEARCHURL%(idx,urllib.quote_plus(title))), "html.parser")
-                posters = soup('div' , {'class': 'fusion-image-wrapper'})
-                if not posters or len(posters) == 0: break
-                for poster in posters:
-                    url = poster.find_all('img')[0].attrs['src']
-                    listItems.append(xbmcgui.ListItem(poster.find_all('h4')[0].get_text(), thumbnailImage=url, path=url))
+                soup  = BeautifulSoup(self.openURL(SEARCHURL%(idx,urllib.quote_plus(title))), "html.parser")
+                items = soup('div' , {'class': 'fusion-post-wrapper'})
+                if not items or len(items) == 0: break
+                for item in items:
+                    url = item('div' , {'class': 'fusion-image-wrapper'})[0].find('img').attrs['src']
+                    label = item('div' , {'class': 'fusion-post-content post-content'})[0].find('a').get_text()
+                    listItems.append(xbmcgui.ListItem(label, thumbnailImage=url, path=url))
```

Comment:
thumbnailImage is deprecated

Reply:
It's still a usable parameter. https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gac31a08def90f50295146753353cb9541

Reply:
Being deprecated means it'll eventually be removed but is still available. This was deprecated in v16 and we are about to reach v18... 
We've been asking everyone to stop using these kwargs so please adjust it in this update

Reply:
https://github.com/xbmc/repo-scripts/pull/842

Reply:
missed it in the last submission. Make sure you remove it on the next update